model/src/main/java/de/danoeh/antennapod/model/feed/FeedPreferences.java

```diff
@@ -211,4 +211,11 @@ public boolean getShowEpisodeNotification() {
     public void setShowEpisodeNotification(boolean showEpisodeNotification) {
         this.showEpisodeNotification = showEpisodeNotification;
     }
+
+    public void setTags(Set<String> tags) {
+        Set<String> newTags = new HashSet<>();
+        newTags.addAll(this.tags);
```

Comment:
This **adds** tags, it doesn't **set** them

Reply:
Yeah, i know. Is this not okay?

Reply:
I think it is only used for testing.